### PR TITLE
[15.0][MIG] mass_mailing_resend: Migration to 15.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 15.0

Please @pedrobaeza and @CarlosRoca13 can you review it?

@Tecnativa TT40296

### CODE REVIEW

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Testing on runboat, the button "Emails sent" is not shown on draft state, so the text saying that you can resend to all removing the recipients in it it's not true. We should restore button visibility in this module, plus maybe adding a test for being sure about the statement in that label (already sent not sent again).

**Comment by victoralmau** [comment]
Changes done.

**Comment by pedrobaeza** [comment]
I think you haven't understood me... The banner is now shown on `sent` state, while it should only be shown after pressing resend (and being on draft state). The problem was that on draft state, no "N Sent" smart-button is shown, so you are not able to remove any of them for resending the mail to them.

**Comment by victoralmau** [comment]
Ok sorry, I think I know what you initially meant.
Changes done.

File: mass_mailing_resend/views/mailing_mailing_views.xml [hunk 1]
```diff
@@ -16,13 +15,13 @@
...
                     class="oe_form_box_info bg-warning oe_text_center"
-                    attrs="{'invisible': ['|', ('state', '!=', 'draft'), ('sent', '=', 0)]}"
+                    attrs="{'invisible': ['|', ('state', '!=', 'draft'),('sent', '=', 0)]}"
                 >
                     <p>
                         <i
                             class="fa fa-info-circle"
                         /> New sending will be done only to not sent/new recipients. If you want to resend again the mass mailing to already sent recipients, click on <b
-                        >Emails Sent</b> smart-button for removing the existing record(s).
+                        >Send</b> smart-button for removing the existing record(s).
```
Comment (pedrobaeza) [inline mass_mailing_resend/views/mailing_mailing_views.xml]:
```suggestion
                        >Sent</b> smart-button for removing the existing record(s).
```

File: mass_mailing_resend/views/mailing_mailing_views.xml [hunk 2]
```diff
@@ -16,13 +15,13 @@
             <header position="after">
                 <div
                     class="oe_form_box_info bg-warning oe_text_center"
-                    attrs="{'invisible': ['|', ('state', '!=', 'draft'), ('sent', '=', 0)]}"
+                    attrs="{'invisible': ['|', ('state', '!=', 'draft'),('sent', '=', 0)]}"
```
Comment (pedrobaeza) [inline mass_mailing_resend/views/mailing_mailing_views.xml]: No need of this diff

File: mass_mailing_resend/i18n/es.po [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+#: model_terms:ir.ui.view,arch_db:mass_mailing_resend.view_mail_mass_mailing_form
+msgid ""
+"<i class=\"fa fa-info-circle\"/> New sending will be done only to not sent/"
+"new recipients. If you want to resend again the mass mailing to already sent "
+"recipients, click on <b>Sent</b> smart-button for removing the "
+"existing record(s)."
+msgstr ""
+"<i class=\"fa fa-info-circle\"/> El nuevo envío se realizará solo a los "
+"destinatarios no enviados/nuevos. Si quiere reenviar otra vez el correo "
+"masivo a destinatarios ya enviados, pulse en el botón <b>Enviar</"
```
Comment (pedrobaeza) [inline mass_mailing_resend/i18n/es.po]:
```suggestion
"masivo a destinatarios ya enviados, pulse en el botón <b>Enviados</"
```

**Review by pedrobaeza — APPROVED** [review]

**Review by CarlosRoca13 — APPROVED** [review]
Test and code review :+1:

**Pull request merged**